src/containers/Header/header.scss

```diff
@@ -137,72 +151,28 @@
     }
 
     &.testnet {
-      background: $testnet;
-
-      .dropdown {
-        border-color: $black;
-        background: $testnet;
-        color: $white;
-
-        /* stylelint-disable no-descending-specificity */
-        .item {
-          background: $testnet;
-        }
-        /* stylelint-enable no-descending-specificity */
-      }
+      @include dropdown-network-item($testnet);
     }
 
     &.devnet {
-      background: $devnet;
-
-      .dropdown {
-        border-color: $black;
-        background: $devnet;
-        color: $white;
+      @include dropdown-network-item($devnet);
+    }
 
-        /* stylelint-disable no-descending-specificity */
-        .item {
-          background: $devnet;
-        }
-        /* stylelint-enable no-descending-specificity */
-      }
+    &.amm {
+      @include dropdown-network-item($amm);
     }
 
     /* stylelint-disable-next-line selector-class-pattern -- Name comes from server designation  */
     &.nft_sandbox {
-      background: $nft-sandbox;
-
-      .dropdown {
-        border-color: $black;
-        background: $nft-sandbox;
-        color: $white;
-
-        /* stylelint-disable no-descending-specificity */
-        .item {
-          background: $nft-sandbox;
-        }
-        /* stylelint-enable no-descending-specificity */
-      }
+      @include dropdown-network-item($nft-sandbox);
 
       .check path {
         fill: $white;
       }
     }
 
     &.sidechain {
-      background: $sidechain;
-
-      .dropdown {
-        border-color: $sidechain;
-        background: $sidechain;
-        color: $black;
-
-        /* stylelint-disable no-descending-specificity */
-        .item {
-          background: $sidechain;
-        }
-        /* stylelint-enable no-descending-specificity */
-      }
+      @include dropdown-network-item($sidechain, $sidechain, $black);
```

Comment:
Is it bad practice to name $sidechain something like $sidechain-color to indicate the type? (Without reading the implementation it's a bit unclear what the variable is supposed to represent - Ex. there's lots of sidechain things it could be related to)

Whatever is best should be applied to the other network-color variables as well.